framework/include/transfers/MultiAppFieldTransfer.h

```diff
@@ -30,37 +34,22 @@ class MultiAppFieldTransfer : public MultiAppTransfer
 
   virtual void initialSetup();
 
+protected:
   /**
-   * Add some extra work if necessary after execute(). For example, adjust the solution
-   * to preserve some physics quality of interest.
+   * Performs the transfer of a variable between two problems if they have the same mesh.
    */
-  virtual void postExecute();
-
-protected:
-  std::vector<VariableName> _from_var_names;
-  std::vector<AuxVariableName> _to_var_names;
-
-  VariableName _from_var_name;
-  AuxVariableName _to_var_name;
+  void transfer(FEProblemBase & to_problem, FEProblemBase & from_problem);
 
-  // If this transfer is going to conserve the physics
-  bool _preserve_transfer;
-  // Postprocessor evaluates an adjuster for the source physics
-  std::vector<PostprocessorName> _from_postprocessors_to_be_preserved;
-  // Postprocessor evaluates an adjuster for the target physics
-  std::vector<PostprocessorName> _to_postprocessors_to_be_preserved;
-
-private:
-  void adjustTransferedSolution(FEProblemBase * from_problem,
-                                PostprocessorName & from_postprocessor,
-                                FEProblemBase & to_problem,
-                                PostprocessorName & to_postprocessor);
-
-  void adjustTransferedSolutionNearestPoint(unsigned int i,
-                                            FEProblemBase * from_problem,
-                                            PostprocessorName & from_postprocessor,
-                                            FEProblemBase & to_problem,
-                                            PostprocessorName & to_postprocessor);
-
-  bool _use_nearestpoint_pps;
+  /**
+   * Performs the transfer of values between a node or element.
+   */
+  void transferDofObject(libMesh::DofObject * to_object,
+                         libMesh::DofObject * from_object,
+                         MooseVariableFEBase & to_var,
+                         MooseVariableFEBase & from_var);
+
+  /// Virtual class defining variables to be transferred
```

Comment:
I'm not sure what this comment means. Virtual class? I believe you mean virtual function.

Reply:
Yes, thank you for catching this.